<?xml version="1.0"?>
<launch>

  <param name="log_file" type="string" value="results4"/>
  <param name="debuger_mode" type="boolean" value="False"/>
    <node pkg="communication_node" type="update_info.py" name="update_info" output="screen" />


</launch>
